src/qt/sendcoinsentry.cpp

```diff
@@ -55,13 +56,36 @@ SendCoinsEntry::SendCoinsEntry(const PlatformStyle *_platformStyle, QWidget *par
     connect(ui->deleteButton, &QToolButton::clicked, this, &SendCoinsEntry::deleteClicked);
     connect(ui->deleteButton_is, &QToolButton::clicked, this, &SendCoinsEntry::deleteClicked);
     connect(ui->deleteButton_s, &QToolButton::clicked, this, &SendCoinsEntry::deleteClicked);
+    connect(ui->messageTextLabel, &QLineEdit::textChanged, this, &SendCoinsEntry::on_MemoTextChanged);
+
+    ui->messageLabel->setVisible(false);
+    ui->messageTextLabel->setVisible(false);
+    ui->iconMessageWarning->setVisible(false);
 }
 
 SendCoinsEntry::~SendCoinsEntry()
 {
     delete ui;
 }
 
+void SendCoinsEntry::on_MemoTextChanged(const QString &text)
+{
+    int maxLength = 256;
```

Comment:
Hard coding numbers like this is wrong coding style, also this is not right number. use get_memo_bytes() function to get memo size limit.